src/Moq.EntityFrameworkCore.Examples/Users/UsersContext.cs

```diff
@@ -7,6 +7,6 @@ public class UsersContext : DbContext
     {
         public virtual DbSet<User> Users { get; set; }
 
-        public virtual DbQuery<Role> Roles { get; set; }
```

Comment:
Deprecated - does not mean deletesd. Still, support should be in the package. Only Obsolete attribute should be added.